target.py

```diff
@@ -0,0 +1,50 @@
+import cv2
+import math
+
+
+class Target:
+    FOV = (50.28, 29.16)
+    RES = (1280, 720)
+    CAM_HEIGHT = 0.15
```

Comment:
I don't think it's a good idea to have the camera constants be copied in every class. Maybe we could pass the camera parameters we need in the constructor?